Jenkinsfile.conda

```diff
@@ -0,0 +1,55 @@
+pipeline {
+    agent any
+    environment {
+        idl_version = "${GIT_TAG_NAME}"
+        xml_version = "4.6.0"
+        dockerImageName = "lsstts/conda_package_builder:latest"
+        container_name = "idl_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
+    }
+
+    stages {
+        stage("Pull Docker Image") {
+            steps {
+                script {
+                    sh """
+                    docker pull ${dockerImageName}
+                    """
+                }
+            }
+        }
+        stage("Create IDL Conda package") {
+            steps {
+                script {
+                    sh """
+                    docker run --name ${container_name} -di --rm -v ${WORKSPACE}/conda:/home/saluser/conda ${dockerImageName}
+                    docker exec -u root -w /tmp ${container_name} sh -c "repoquery --disablerepo="*" --enablerepo="lsst-ts" -a \
+                        |grep -v _test- |grep -v python-0 |grep -v OpenSplice |xargs yum install -y"
+                    docker exec --env TS_XML_VERSION=v${xml_version} ${container_name} sh -c "cd ~/conda && source ~/miniconda3/bin/activate && conda build ."
```

Comment:
Instead of injecting environment variables (e.g. `TS_XML_VERSION`) at `exec` I think it is better to do it and `run` time.

Reply:
Fixed in commit a27266d0ca07c4a24761d164fcef795645fe1aa4.